### PR TITLE
Oceans: deep blue background for specific levels

### PR DESCRIPTION
**Pull request by breville**
Back in https://github.com/code-dot-org/code-dot-org/pull/37441, the Oceans script got a deep blue background.

Now that the Oceans levels are appearing in other scripts that also contain non-Oceans levels, we want the deep blue background, but only for the specific levels. 

Support for per-script backgrounds has been removed, since Oceans was the only script using this feature.

### before

Screen Shot 2021-07-02 at 10 28 54 AM

### after

Screen Shot 2021-07-02 at 10 30 17 AM

### CODE REVIEW

**Review by davidsbailey — APPROVED** [review]
this looks good, thanks for adjusting!

File: dashboard/app/controllers/script_levels_controller.rb [hunk 1]
```diff
@@ -157,7 +157,7 @@ def show
     @level = select_level
     return if redirect_under_13_without_tos_teacher(@level)
 
-    @body_classes = @script_level.script.background
+    @body_classes = @level.properties['background'] || @script_level.script.background
```
Comment (maddiedierker) [inline dashboard/app/controllers/script_levels_controller.rb]: should we remove the `background` property from script.rb with this change? do you know if any other scripts use it?

Comment (bethanyaconnor) [reply dashboard/app/controllers/script_levels_controller.rb]: Querying my local database, only `oceans` uses the background property

**Review by bethanyaconnor — APPROVED** [review]

File: dashboard/app/controllers/script_levels_controller.rb (hunk 1 above)
Comment (breville) [reply dashboard/app/controllers/script_levels_controller.rb]: I was thinking other scripts might still want this feature, but given that none had, I've now removed the support.

**Review by maddiedierker — APPROVED** [review]
🐟 🐠

**Pull request merged**